docs/sources/installation/windows.md

```diff
@@ -10,9 +10,9 @@ weight = 400
 
 # Installing on Windows
 
-Description | Download
------------- | -------------
-Latest stable package for Windows | [x64](https://grafana.com/grafana/download?platform=windows)
+[Download the latest stable package for Windows.](https://grafana.com/grafana/download?platform=windows)
```

Comment:
Do we not support Windows 32-bit?

Reply:
No, I don't think so

Reply:
Remember when I asked you jokers about software requirements? I will add this detail to requirements.md.